apm/package.json

```diff
@@ -6,6 +6,6 @@
     "url": "https://github.com/atom/atom.git"
   },
   "dependencies": {
-    "atom-package-manager": "2.5.0"
+    "atom-package-manager": "github:DeeDeeG/apm#support_visual_studio_2017_and_2019"
```

Comment:
Eventually, make a pull request to apm. If they were slow to merge, we will create our `apm` fork in this organization.

Reply:
https://github.com/atom/apm/pull/892